server/server.js

```diff
@@ -1,15 +1,141 @@
 const express = require("express");
+const cors = require("cors");
+const moment = require("moment");
+
+
+
 const app = express();
-const port = process.env.PORT || 5000;
 
+app.use(express.json());
+app.use(cors());
+
+const port = process.env.PORT || 5000;
```

Comment:
Try and use `??` over `||` in most cases; `||` triggers on falsey values which is often not the intention when it's used.